wlinux-setup

```diff
@@ -50,10 +50,11 @@ function ByeMessage {
 function InstallMenu {
 
 MenuChoice=$(
-whiptail --title "wlinux-setup" --checklist --separate-output "\nHand-curated add-ons [SPACE to select, ENTER to confirm]:" 22 99 15 \
+whiptail --title "wlinux-setup" --checklist --separate-output "\nHand-curated add-ons [SPACE to select, ENTER to confirm]:" 22 99 16 \
     "LANGUAGE" "Change default language and keyboard setting in WLinux" off \
     "EXPLORER" "Enable right-click on folders in Windows Explorer to open them in WLinux  " off \
     "SHELLS" "Install and configure zsh, csh, and fish" off \
+    "SHELL-OPTS" "Install optimised shell startup files" off \
```

Comment:
I don't like the title looks like something I wouldn't click. I will think in proposals

Reply:
Ah right yeah this was something I meant to (but forgot :sweat_smile:) to include in the PR comment. The text I've written for user-prompts in this could definitely do with some proof reading and when I come back to it after looking at Circle Ci there's definitely some of my own changes I'll look at submitting